.github/workflows/build.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         run: |
           yarn
           pipx install black[colors]
-          pipx install cairo-lang==0.8.2
+          pipx install cairo-lang==0.8.2.1
```

Comment:
The cairo lang version appears twice in this file, would be best to keep them in sync